.github/workflows/bsim-tests.yaml

```diff
@@ -178,7 +178,6 @@ jobs:
 
       - name: Merge Test Results
         run: |
-          pip install junitparser junit2html
```

Comment:
Looks like you just dropped them as I can't see any step where you install the requirements?

Reply:
those are already in the docker image, no need to install them